rclpy/test/test_node.py

```diff
@@ -337,6 +337,41 @@ def test_count_publishers_subscribers(self):
         with self.assertRaisesRegex(ValueError, 'is invalid'):
             self.node.count_publishers('42')
 
+    def test_count_clients_services(self):
+        short_service_name = 'add_two_ints'
+        fq_service_name = '%s/%s' % (TEST_NAMESPACE, short_service_name)
+
+        self.assertEqual(0, self.node.count_clients(fq_service_name))
+        self.assertEqual(0, self.node.count_services(fq_service_name))
+
+        self.node.create_client(GetParameters, short_service_name)
+        self.assertEqual(1, self.node.count_clients(short_service_name))
+        self.assertEqual(1, self.node.count_clients(fq_service_name))
+        self.assertEqual(0, self.node.count_services(short_service_name))
+        self.assertEqual(0, self.node.count_services(fq_service_name))
+
+        self.node.create_service(GetParameters, short_service_name, lambda req: None)
+        self.assertEqual(1, self.node.count_clients(short_service_name))
+        self.assertEqual(1, self.node.count_clients(fq_service_name))
+        self.assertEqual(1, self.node.count_services(short_service_name))
+        self.assertEqual(1, self.node.count_services(fq_service_name))
+
+        self.node.create_client(GetParameters, short_service_name)
+        self.assertEqual(2, self.node.count_clients(short_service_name))
+        self.assertEqual(2, self.node.count_clients(fq_service_name))
+        self.assertEqual(1, self.node.count_services(short_service_name))
+        self.assertEqual(1, self.node.count_services(fq_service_name))
```

Comment:
what about creating multiple server case?

Reply:
added. sorry for late :)